src/main/java/com/powsybl/ws/commons/SecuredInputStream.java

```diff
@@ -11,14 +11,14 @@
  */
 public class SecuredInputStream {
     private final int maxEntries;
-    private final long maxSize;
+    private final long maxUncompressedSize;
```

Comment:
```suggestion
    // This attribute defines the maximum for the total of the sizes of the uncompressed entries. This limit is defined to avoid zip/tar bombs.
    private final long maxUncompressedSize;
```